Service/utils.js

```diff
@@ -40,16 +41,21 @@ const checkAttributes = (endUserReq, experiment, next) => {
     try {
         const geo = getLocation(getClientIP(endUserReq));
         const {browser, device} = getBrowserDevice(endUserReq);
+        const customAttributes = endUserReq.customAttributes;
         if (geo && browser && device) {
-            const result =
-                geo.country === experiment.testAttributes.location[0] &&
-                browser === experiment.testAttributes.browser[0] &&
-                device === experiment.testAttributes.device[0];
-            const customAttributes = endUserReq.customAttributes;
```

Comment:
what if we don't have custom attributes?

Reply:
then it should be null or empty